### PR TITLE
Updated make_request per iss261

### LINKED ISSUE #261: Improve documentation for sending body in make_requests
Hey team, Boxer here :) submitting PR for this shortly.

The helper function here:
https://github.com/box/box-python-sdk/blob/1.5/README.rst#other-requests

Demonstrates how to specify a method and URL for a custom API call. However, it is not clear how to send the body.

Looking closer at the docs:
https://github.com/box/box-python-sdk/blob/f77cb41afe91f1f1d9b7d6b0044ae5eee358c977/boxsdk/client/client.py#L298

It looks like it takes **kwargs, which is some sort of parameter handling. As a beginner, it's not easy to understand how to send a body for a custom API call.

See PR for suggested doc updates...

### PR DESCRIPTION
**Pull request by jpan-box**
Fixes #261

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jpan-box has signed the CLA. Thanks for the pull request!

**Comment by jpan-box** [comment]
Fixed formatting errors. Should be correct now.

File: README.rst [hunk 1]
```diff
@@ -302,6 +302,20 @@ supported by the SDK. You can still use these endpoints by using the ``make_requ
...
+.. code-block:: python
+
+    # https://developer.box.com/reference#update-terms-of-service-user-status
+    # Updates a user's ToS status
+
+    # JSONify the body
+    body = json.dumps({"is_accepted":true})
+
+    # Pass body as "data" argument
+    data = body
```
Comment (anthonywee) [inline README.rst]: Why do we set `data = body` here?

Comment (jasonpan12) [reply README.rst]: @anthonywee That seems to be what make_request needs to make passing data as body work properly. One of the most recent messages in our internal slack channel for the Python SDK refers to setting data = body, or data = whatever object contains the jsonified data. Maybe there's a better way to communicate that?

Comment (jpan-box) [reply README.rst]: sorry, that's my personal ^ account. This is from my actual box github account.

**Pull request closed without merging**